lib/generators/action_client/templates/action_client.rb.tt

```diff
@@ -0,0 +1,7 @@
+class <%= class_name %>Client < ApplicationClient
```

Comment:
I don't remember where we ended up with setting up the default url here.

Reply:
I think we had a plan to generate the `default` call:

```ruby
class ArticlesClient < ApplicationClient
  default configuration.url
end
```

Reply:
I wonder if we'd be better off generating YML files with both `url` and `headers` keys, and then passing them to `default` when `ActionClient::Base` is descended from:

```yml
url: "https://example.com"
headers:
  Content-Type: "application/json"
```

That way, the generators establish the pattern of declaring global headers in the YML, instead of establishing the `default` calls.